src/lib/orm/entities/RpgUserItemEntity.ts

```diff
@@ -0,0 +1,32 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { RpgBattleEntity } from './RpgBattleEntity';
+import { RpgItemEntity } from './RpgItemEntity';
+import { RpgUserEntity } from './RpgUserEntity';
+
+@Entity('rpg_user_item', { schema: 'public' })
+export class RpgUserItemEntity extends BaseEntity {
+
+	@PrimaryGeneratedColumn({ type: 'bigint' })
+	public id?: string;
```

Comment:
id cannot be undefined

type: 'bigint' but field type string?

Reply:
`PrimaryGeneratedColumn` doesn't support `transformer`, and `pg` returns `bigint` as `string`s.